web_services/foxx/brightid5/initdb.js

```diff
@@ -98,13 +99,45 @@ function v5() {
   }
 }
 
-const upgrades = ['v5'];
+function v5_3() {
+  const usersColl = arango._collection('users');
+  const connectionsColl = arango._collection('connections');
+  const timestamp = Date.now();
+
+  connectionsColl.all().toArray().forEach(conn => {
+    const key1 = conn._from.replace('users/', '');
+    const key2 = conn._to.replace('users/', '');
+    if (conn.timestamp < 1597276800000) {
```

Comment:
>  It won't affect the "connected with friends" verification, will it?

No, those connections that we are trying to set to `already known` by default do not result in those users get `SeedConnectedWithFriend` verification. They should connect to a single seed in with short gap to get that verification.

> Do we need to worry about setting some connections to "already known"?

I'm not sure what you think we may need to worry about.

> It shouldn't be a problem to let people label some of their connections as "already known" by clicking on their profiles.
it's not problem, but I guess in this way we make this simpler for users because connections that are created 2 months ago are mostly `already known` connections. So people can simply just set a few connections level to `just met` instead of updating lots of connections to `already known`

Reply:
> No, those connections that we are trying to set to `already known` by default do not result in those users get `SeedConnectedWithFriend` verification. They should connect to a single seed in with short gap to get that verification.

What I mean is would it hurt someone's chances to get `SeedConnectedWithFriend` if they only had `just met` connections?

Reply:
> it's not problem, but I guess in this way we make this simpler for users because connections that are created 2 months ago are mostly `already known` connections. So people can simply just set a few connections level to `just met` instead of updating lots of connections to `already known`

>it's not problem, but I guess in this way we make this simpler for users because connections that are created 2 months ago are mostly already known connections. So people can simply just set a few connections level to just met instead of updating lots of connections to already known

I think it's the opposite--that most connections made 2 months ago are "just met." Maybe it's not too important which we choose.

Reply:
> > it's not problem, but I guess in this way we make this simpler for users because connections that are created 2 months ago are mostly `already known` connections. So people can simply just set a few connections level to `just met` instead of updating lots of connections to `already known`
> 
> > it's not problem, but I guess in this way we make this simpler for users because connections that are created 2 months ago are mostly already known connections. So people can simply just set a few connections level to just met instead of updating lots of connections to already known
> 
> I think it's the opposite--that most connections made 2 months ago are "just met." Maybe it's not too important which we choose.

This is true for you and few active seeds. But not for most people.

New connections mostly created in parties so they should be `just met` by default.
But we can decide if we set old ones by default to `just met` just like new connections or set them `already know` by default.